examples/LossMinimization/loss_minimization.jl

```diff
@@ -1,122 +1,201 @@
-using Distributions
-using LinearAlgebra
-using Random
-using Plots
-using EnsembleKalmanProcesses.EnsembleKalmanProcessModule
-using EnsembleKalmanProcesses.ParameterDistributionStorage
-
-# Seed for pseudo-random number generator for reproducibility
+# # Minimization Loss
```

Comment:
Is it "Minimization Loss" or "Loss Minimization" ???

Reply:
I rephrased to "Minimization of Loss function"